.env.development

```diff
@@ -14,13 +14,11 @@ REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 USER_INFO_COOKIE_NAME='edx-user-info'
 SITE_NAME=localhost
-
 DATA_API_BASE_URL='http://localhost:8000'
 // LMS_CLIENT_ID should match the lms DOT client application id your LMS containe
 LMS_CLIENT_ID='login-service-client-id'
-SEGMENT_KEY=null
+SEGMENT_KEY=''
```

Comment:
This is an important little change - it turns SEGMENT_KEY from a string, "null", to a falsy empty string.